.github/workflows/tests.yaml

```diff
@@ -52,7 +68,7 @@ jobs:
 
       - name: Unit Tests
         run: |
-          pytest tests
+          pytest -vvv tests
```

Comment:
Does this actually provide useful information? Or can it be removed?

Reply:
I just did this for debugging and forgot to remove it. The benefit is it forces pytest to show the full context in error messages.